packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,118 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('no response example is requested specifically, and a response example is defined', () => {
+          it('should return the first example', () => {
```

Comment:
```suggestion
          it('returns the first example', () => {
```

Trying to be more assertive with our assertions. Clean up any old should's you find laying around 🗡

Reply:
Roger!